opam-packages-conversion/bin/config.py

```diff
@@ -0,0 +1,182 @@
+import os.path
+
+__dir__ = os.path.dirname(os.path.realpath(__file__))
+
+os.environ['PATH'] = '%s:%s' % (__dir__, os.environ['PATH'])
+
+GH_ORG_NAME = 'esy-ocaml-test'
+NPM_SCOPE = 'opam'
+
+OPAM_PACKAGES = os.path.join(os.path.dirname(__file__), '..', 'opam-repository', 'packages')
+
+OPAM_PACKAGES_BLACKLIST = {
+    "camlp4.4.03+system",
+    "camlp4.4.02+system",
+    "camlp4.4.04+system",
+    "inotify.2.2", # doesn't support mac
+    "inotify.2.3",
+    "ocaml-data-notation.0.0.9",
+    "lwt.2.6.0", # temp
+    "ppx_tools.5.0+4.03.0"
+    }
+
+ESY_EXTRA_DEP = {
+    "camomile": ['cppo', 'camlp4']
+}
+
+OPAM_DEPOPT_BLACKLIST = {
+    "conf-libev",
+    "lablgtk",
+    "ssl",
+    "mirage-xen-ocaml",
+    "tyxml",
+    "reactiveData",
+    "deriving",
+    "ocamlbuild",
+    }
+
+GH_USER = os.environ.get('GH_USER', '')
+GH_TOKEN = os.environ.get('GH_TOKEN', '')
+
+cleanup = '(make clean || true)'
+opam_install = '(opam-installer --prefix=$cur__install || true)'
+
+drop_beta_from_version = lambda version: version.replace('-beta', '')
+
+OVERRIDE = {
+    'ocp-build': {
+        'version': drop_beta_from_version
+    },
+    'typerex-build': {
+        'version': drop_beta_from_version,
+        "build": [
+            "./configure --prefix $cur__install",
+            "make",
+            "make install",
+            "(opam-installer --prefix=$cur__install || true)"
+        ],
+    },
+    'ocamlbuild': {
+        'build': 'true',
+    },
+    'ocamlfind': {
+        'build': [
```

Comment:
Since ocamlfind has an override here, which takes precedence? This override, or the custom `_build` script in the fork?

https://github.com/esy-ocaml/ocamlfind/commit/38052519804fcb485e21bde0aca6b89ac9fab4d1#diff-876c319af3a4cb0cb8b20fa1913434c6R1

If the `_build` script takes precedence, do we get to remove this override?